vulnfeeds/cmd/cperepos/main.go

```diff
@@ -77,10 +78,43 @@ const (
 var (
 	Logger utility.LoggerWrapper
 	// These repos should never be considered authoritative for a product.
+	// TODO(apollock): read this from an external file
 	InvalidRepos = []string{
+		"https://github.com/abhiunix/goo-blog-App-CVE",
```

Comment:
Do we need all of these, if you already have the Regex to look for CVEs in the repo name?

Reply:
My regex is moderately conservative to avoid potential false positives